src/js/views/patients/patient/sidebar/patient-sidebar.scss

```diff
@@ -1,15 +1,27 @@
 .patient-sidebar {
-  padding: 40px 24px 0;
+  border-left: 1px solid #e6e6e6;
+  padding: 54px 24px 0;
```

Comment:
Padding here is adjusted her to get the desired spacing between the icon/ellipsis and patient name below it.

The padding is kind of crazy here because the icon and ellipsis are using `position: absolute`. 

Would it be better to not use `position: absolute`? And position them as a normal `flex` element like we did in the worklist patient sidebar?

I think it would make the CSS styling for it a little simpler.

Reply:
Hmm yeah this is problematic.  Any pixel size that isn't divisible by 4px should be incorrect or commented as to why it is specifically needed.  If 52 or 56 cannot work it probably indicates some other styling issue somewhere.